### ISSUE TITLE
Add image/kernel/repo info to hardware/SBC section

### ISSUE THREAD

**Issue by MichaIng** [MichaIng/DietPi-Docs#365]
It's interesting to read details about history and features of SBC devices, but users who land at our docs have their hardware already, hence know about those details already or do not find it relevant. What is relevant when dealing with DietPi images is:
- Which SBC-specific APT repository the image is shipped with.
- Related to that, which kernel version/source.

We have a short pre-image info in the DietPi header, but that alone is a bit short in most cases.

Specifically it could look like this:
- APT repositories:
  - Debian: https://deb.debian.org/debian/
  - Kernel/firmware: https://dietpi.com/meveric/
- Pre-installed kernel/firmware packages: `linux-image-4.14-armhf-odroid-xu4`
- Kernel/firmware repository maintainer: https://forum.odroid.com/viewtopic.php?t=30552
- Kernel sources: https://github.com/hardkernel/linux/tree/odroidxu4-4.14.y

**Comment by Joulinar** [comment]
ok yes that's how I had it in mind. At least to get an overview. Most of our user are not aware that DietPi not an own OS. This can be used to show where the image used is coming from 😃

**Comment by MichaIng** [comment]
Yeah, I mean when running `apt update` the repositories are shown right there. But it requires at least some understanding about how GNU/Linux distributions/images are composed, the difference between distros that maintain their own APT/RPM/whatever package repository (Debian, Ubuntu, Fedora, Arch, ...), those which use other's repositories for major part of the OS, while concentration on kernel/firmware to get it running on particular hardware (Armbian, Raspberry Pi OS 64-bit, generally SBC manufacturer images, Meveric's), and those which even use others kernel/firmware repositories and concentrate more on userspace features, like us 😃, Lysmarine, Picroft, SteamOS and such. Quite a jungle when being new to Linux.
